player/playloop.c

```diff
@@ -1211,6 +1211,12 @@ static void handle_eof(struct MPContext *mpctx)
     }
 }
 
+static void handle_clipboard_updates(struct MPContext *mpctx)
+{
+    if (mp_clipboard_data_changed(mpctx->clipboard))
+        mp_notify_property(mpctx, "clipboard");
```

Comment:
Could you remind me what would be the use-case for monitoring the clipboard in mpv? I always though it would rather be used to past text / image data on demand by user, rather than monitoring it constantly and acting. I would even argue that I don't want my media player to act depending on my clipboard content automatically. That's in no means rejecting the idea, but I would like to know when would that be useful.

Reply:
> Could you remind me what would be the use-case for monitoring the clipboard in mpv? 
Could be used for auto playing Links(like YouTube) while copying links or so, some other programs do that too I think
Never personally used it but nice to have I guess. And: and > "properties are generally expected to be observable."